data/STRchive-database.json

```diff
@@ -2759,61 +4249,96 @@
         "stop_hg19":23790711,
         "start_t2t":17522488,
         "stop_t2t":17522518,
-        "notes_t2t":null,
         "id":"OPMD_PABPN1",
         "disease_id":"OPMD",
         "gene_strand":"+",
-        "reference_motif_reference_orientation":"GCN",
-        "pathogenic_motif_reference_orientation":"GCN",
-        "pathogenic_motif_gene_orientation":"CNG",
-        "benign_motif_reference_orientation":null,
-        "benign_motif_gene_orientation":"",
-        "unknown_motif_reference_orientation":null,
-        "unknown_motif_gene_orientation":"",
+        "reference_motif_reference_orientation":[
+            "GCN"
+        ],
+        "pathogenic_motif_reference_orientation":[
+            "GCN"
+        ],
+        "pathogenic_motif_gene_orientation":[
+            "CNG"
+        ],
+        "benign_motif_reference_orientation":[],
+        "benign_motif_gene_orientation":[],
+        "unknown_motif_reference_orientation":[],
+        "unknown_motif_gene_orientation":[],
         "disease":"Oculopharyngeal muscular dystrophy",
         "gene":"PABPN1",
         "flank_motif":null,
         "locus_structure":"(GCN)*",
-        "Inheritance":"AD\/AR",
+        "inheritance":[
+            "AD/AR"
```

Comment:
this seems like a reasonable place to expand into the list

Reply:
Looks like there's one entry, `OPMD_PABPN1`, where the split was missed: ` "inheritance": ["AD/AR"],`